accounts/admin.py

```diff
@@ -188,7 +188,24 @@ def delete_preserve_sounds(self, request, obj):
 
     change_actions = ('full_delete', 'delete_include_sounds', 'delete_preserve_sounds', )
 
+    def save_model(self, request, obj, form, change):
```

Comment:
Wouldn't it be better to do this in a `User` `pre_save` signal so that we make sure that it will be ok even if we change a username through the console (which could happen)?

Reply:
I thought that if we only were going to change this in the admin then it didn't make sense every time we do a change in a user object to check if the username changes. But after this a new issue was created regarding allowing the user to change the username so now it might make sense to do it in a pre save.

Reply:
Well, given that we almost never change user objects (only profiles) then I think we're fine.